apps/teams-test-app/src/components/SearchAPIs.tsx

```diff
@@ -27,7 +27,7 @@ const RegisterHandlers = (): React.ReactElement =>
       };
       setResult('register handlers');
 
-      search.registerHandlers(onChange, onClosed, onExecute);
+      search.registerHandlers(onClosed, onExecute, onChange);
       return 'recieved';
```

Comment:
nit: spelling

Reply:
Oh good catch! Since you're here @jobutle,  would you mind please correcting line 31 to  'received'?